core/lib/workarea.rb

```diff
@@ -182,6 +182,14 @@ def self.define_content_block_types(&block)
     require_dependency Workarea::Core::Engine.root.join('app', 'models', 'workarea', 'content', 'block_type_definition')
     Content::BlockTypeDefinition.new.instance_eval(&block)
   end
+
+  # Use this deprecation to warn about the next minor release.
+  #
+  # @return [ActiveSupport::Deprecation]
+  #
+  def self.deprecation
+    @deprecation ||= ActiveSupport::Deprecation.new('3.6', 'Workarea')
```

Comment:
I feel like passing the version to the `deprecation` method would help us grok for deprecations more easily, much like we do for `TODO v3.6` 

```ruby
def self.deprecation(version)
  @deprecation ||= ActiveSupport::Deprecation.new(version, 'Workarea')
end
```

That way we're not always forgetting  to bump this version in this method every minor, and we can more easily `ack "deprecation('3.6')"` at the same time we `ack 'TODO 3.6'`.

Just a thought.

Reply:
Loving this standardization tho. I always find myself looking up how we do this when I need to deprecate stuff.